api/controllers/main.js

```diff
@@ -1,12 +1,19 @@
 const fs = require('fs');
 const path = require('path');
+const SiteWideErrorLoader = require('../services/SiteWideErrorLoader');
 
 const manifestPath = path.join(__dirname, '..', '..', 'webpack-manifest.json');
 const webpackAssets = JSON.parse(fs.readFileSync(manifestPath, 'utf-8'));
 
 module.exports = {
   index(req, res) {
+    let siteWideError = { display: false };
+    if (req.session.authenticated) {
+      siteWideError = SiteWideErrorLoader.loadSiteWideError();
+    }
+
     res.render('index.html', {
+      siteWideError,
```

Comment:
heck, maybe you could just get rid of the `display` property, and return `null` from `loadSiteWideError()` if the `body` (and optionally, `header`) is not present.